x-pack/plugins/triggers_actions_ui/public/application/sections/rule_details/components/rule_definition.tsx

```diff
@@ -0,0 +1,263 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { useState, useEffect } from 'react';
+import {
+  EuiText,
+  EuiSpacer,
+  EuiButtonEmpty,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiPanel,
+  EuiTitle,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { RuleDefinitionProps } from '../../../../types';
+import { RuleType, useLoadRuleTypes } from '../../../..';
+import { useKibana } from '../../../../common/lib/kibana';
+import { hasAllPrivilege, hasExecuteActionsCapability } from '../../../lib/capabilities';
+import { NOTIFY_WHEN_OPTIONS } from '../../rule_form/rule_notify_when';
+import { RuleActions } from './rule_actions';
+import { RuleEdit } from '../../rule_form';
+
+const OBSERVABILITY_SOLUTIONS = ['logs', 'uptime', 'infrastructure', 'apm'];
+
+export const RuleDefinition: React.FunctionComponent<RuleDefinitionProps> = ({
+  rule,
+  actionTypeRegistry,
+  ruleTypeRegistry,
+  onEditRule,
+}) => {
+  const {
+    application: { capabilities },
+  } = useKibana().services;
+
+  const [editFlyoutVisible, setEditFlyoutVisible] = useState<boolean>(false);
+  const [ruleType, setRuleType] = useState<RuleType>();
+  const { ruleTypes, ruleTypeIndex } = useLoadRuleTypes({
+    filteredSolutions: OBSERVABILITY_SOLUTIONS,
+  });
+
+  useEffect(() => {
+    if (ruleTypes.length && rule) {
+      const matchedRuleType = ruleTypes.find((type) => type.id === rule.ruleTypeId);
+      setRuleType(matchedRuleType);
+    }
+  }, [rule, ruleTypes]);
+
+  const getRuleConditionsWording = () => {
+    const numberOfConditions = rule?.params.criteria ? (rule?.params.criteria as any[]).length : 0;
+    return (
+      <>
+        {numberOfConditions}&nbsp;
+        {i18n.translate('xpack.triggersActionsUI.ruleDetails.conditions', {
+          defaultMessage: 'condition{s}',
+          values: { s: numberOfConditions > 1 ? 's' : '' },
+        })}
```

Comment:
nitpick: I know this is moved over from existing code but I wonder if this gets translated correctly? IIRC some languages have the number first, some have the number last.

I'm thinking something like:

```
defaultMessage: '{numOfConditions} condition{s}`
```

maybe I'm wrong and this translates fine, either way its not a huge deal, totally up to you if you want to address this at all.

Reply:
Yea ive seen something like this done in kibana:
```
i18n.translate('core.euiPrettyDuration.lastDurationSeconds', {
  defaultMessage: 'Last {duration, plural, one {# second} other {# seconds}}',
  values: { duration },
});
```
if you do a search for `plural` you can see where its used, I'm not sure if it accounts for "0 conditions" but it seems like it should.

Reply:
Thanks @JiaweiWu. I used the suggested approach. Yes, it says `0 conditions` :)